packages/ui/src/assets/constant.ts

```diff
@@ -119,3 +119,34 @@ export const NavBarData: NavBarDataType[] = [
     image: '/profile-pic.png',
   },
 ]
+export const sidebarData = {
+  logoSrc: '/default-pic.png',
+  title: '#39 Water Tower Place',
+  links: [
+    { href: '/', label: 'Dashboard', iconName: 'letter' },
+    {
+      href: '/clients',
```

Comment:
When you click on Clients menu item, it will navigate to all clients page and high light the All sub item.